itest/godot/ManualFfiTests.gd

```diff
@@ -29,6 +29,15 @@ func test_export():
 	obj.int_val = 5
 	assert_eq(obj.int_val, 5)
 
+	obj.int_val_get = 5
+	assert_eq(obj.int_val_get, 10)
+
+	obj.int_val_set = 5
+	assert_eq(obj.int_val_set, 10)
+
+	obj.int_val_get_set = 5
+	assert_eq(obj.int_val_get_set, 5)
```

Comment:
Maybe to separate the operation from the read/writability, we could name them:
```rs
int_val_read
int_val_write
int_val_rw
```
This would avoid awkward names like `get_int_val_get_set` in favor of `get_int_val_rw`.

Reply:
`int_val_get` is not currently read-only though. It has a default setter, and vice versa.

This argument becomes invalid if we model it after gdnative, which I think is a good idea.